test/JDBC/upgrade/13_4/preparation/BABEL-3267-vu-prepare.mix

```diff
@@ -28,6 +28,18 @@ go
 -- double quote
 create table [T3267"d] (c varchar(10));
 insert into [T3267"d] values ('success');
+go
```

Comment:
No no, the output is going to be different in case of normal test and in case of upgrade. So that's why I have separately added normal test in ISC-Views.sql file.

Reply:
I mean same across all upgrade cases. Will same prepare script will work on 13.6, 13.7 and 14.3. If yes, we just need to add single prepare file with corresponding entry to schedule file

Reply:
Seems like `13_5__preparation` is not the same as `13_6__preparation`.

It is more like a test framework question, but what if we manage 10 different versions and 8 of them use the same prepare and/or verification scripts. Do we need to write 10 different files?

Reply:
Removed irrelevant upgrade test files.